api_loader.go

```diff
@@ -139,6 +128,29 @@ func processSpec(referenceSpec *APISpec,
 		return &chainDef
 	}
 
+	pathModified := false
+	for {
+		hash := generateDomainPath(referenceSpec.Domain, referenceSpec.Proxy.ListenPath)
+		if v, e := ListenPathMap.Get(hash); !e || v.(int) < 2 {
```

Comment:
The `< 2` is also ported from the original code. It's the count of API specs with said hash. So 1 means no dups.